File: day3.php
<?php
$lines = explode("\n", file_get_contents('input3.txt'));
$possibles = 0;
foreach($lines as $line) {
    list($a,$b,$c) = explode('  ', trim($line));
    if($a + $b > $c && $c + $a > $b && $c + $b > $a) {
        $possibles++;
    }
}
echo $possibles;

File: day3.exs
{:ok, content} = File.read("input3.txt")
lines = String.split(content, "\r\n", trim: true)

result = Enum.reduce(lines, 0, fn(line, acc) ->
  parts = String.split(line, "  ", trim: true)
  a = Enum.at(parts,0) |> String.strip() |> String.to_integer()
  b = Enum.at(parts,1) |> String.strip() |> String.to_integer()
  c = Enum.at(parts,2) |> String.strip() |> String.to_integer()
  if a + b > c && a + c > b && b + c > a, do: acc + 1, else: acc
end)

IO.puts result
